.github/workflows/test-namespace-arm64-linux.yaml

```
@@ -0,0 +1,49 @@
+# Build on runners from https://namespace.so
+name: test-namespace-arm64-linux
```

I think `aarch64` is official name for the architecture [these days](https://www.phoronix.com/news/MTY5ODk). So you should consider:
```suggestion
name: test-namespace-aarch64-linux
```
Similarly for `bazel-build-arm64-linux` and `./build/pocket-ic-server-arm64-linux` below.

---

OTOH I think they can be used interchangeably. So I don't really care I guess.

---

And `arm64` is faster and easier to type 😂